widgetssdk/src/main/java/com/glia/widgets/locale/StringKeyPair.kt

```diff
@@ -0,0 +1,53 @@
+package com.glia.widgets.locale
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
```

Comment:
why not make these classes internal instead of adding the @hide tag?

Reply:
They are used in java public class. So I first need to convert it to Kotlin then I will be able to make it also internal because it is not supposed to be public and then I will be able to make these classes internal as well. Plan to do that but in a separate PR.